quickstart/101-machine-learning/workspace.tf

```diff
@@ -0,0 +1,57 @@
+# Dependent resources for Azure Machine Learning
+resource "azurerm_application_insights" "default" {
+  name                = "${var.name}-${var.environment}-ain"
```

Comment:
This should align with Azure naming conventions mentioned above and be a prefix of "appi-" instead.